mlrun/datastore/base.py

```diff
@@ -160,8 +160,8 @@ def get_spark_options(self, path=None):
 
     @staticmethod
     def _is_directory_in_range(
-        start_time: datetime.datetime,
-        end_time: datetime.datetime,
+        start_time: Union[datetime.datetime, None],
```

Comment:
Not sure why it doesn't let me comment on both lines at the same time, but the conventional way of doing this is
```suggestion
        start_time: Optional[datetime.datetime],
```